packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -76,10 +76,9 @@ const TutorialStore = types
         try {
           const response = yield tutorials.getAttachedImages({ id: tutorial.id })
           const { media } = response.body
-          self.setMediaResources(media)
+          if (media && media.length > 0) self.setMediaResources(media)
         } catch (error) {
           console.error(error)
```

Comment:
Why is the error state removed here? Are you ignoring 500 errors etc. and just displaying the tutorial without attached media if Panoptes is broken/unavailable?

Reply:
Yep, we don't care if the image request fails. This is the current behavior in PFE. The `catch` there just returns an empty array and doesn't even log the error at all. I think if at some point we setup an error logging service, we'll want to add that here, but we don't want to stop the tutorial from rendering.